Sources/GryphonLib/KotlinTranslator.swift

```diff
@@ -655,6 +655,8 @@ public class KotlinTranslator {
 		let parameterStrings = try functionDeclaration.parameters
 			.map { try translateFunctionDeclarationParameter($0, withIndentation: indentation) }
 
+			let singleExpressionStatement = self.isSingleExpressionFunction(functionDeclaration.statements, returnTypeString: returnTypeString)
```

Comment:
Nitpick: is this indented one extra level?

Reply:
You're right. I was fixing my indentation (because I'm using spaces normally) and that must have slipped through.